wordbook/window.py

```diff
@@ -229,6 +230,7 @@ def threaded_search(self, pass_check=False):
 
         if status == "done":
             self.__page_switch("content_page")
+            self._content_scroll.get_vadjustment().set_value(0)
```

Comment:
Better to move this to `__page_switch` with a check if the page name is `content_page`. And also might be better to use `GLib.idle_add` because it's in a thread.

Reply:
Done.